python/python/glide/async_commands/server_modules/json.py

```diff
@@ -139,6 +139,57 @@ async def get(
     return cast(bytes, await client.custom_command(args))
 
 
+async def arrinsert(
+    client: TGlideClient,
+    key: TEncodable,
+    path: TEncodable,
+    index: int,
+    values: List[TEncodable],
+) -> TJsonResponse[int]:
+    """
+    Inserts one or more values into the array at the specified `path` within the JSON document stored at `key`, before the given `index`.
+
```

Comment:
missing the doc link.
```suggestion
See https://redis.io/docs/latest/commands/json.arrinsert/ for more details.
```
Apparently, there is no json document on `valkey.io`.

Reply:
we decided to remove it for now, I'll see what we will do on feature freeze since its not a big deal for now